README.md

```diff
@@ -1 +1,85 @@
-# P4_Project
\ No newline at end of file
+# STEP (Step To Effective Programming)
+
+SUMMARY
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Dependencies
+
+The project is dependant on the [Arduino-CLI](https://arduino.github.io/arduino-cli/0.21/installation/).
+
+### Installation
+
+1. Clone the repository
+
+```sh
+git clone https://github.com/KarmaKamikaze/STEP.git
+```
+
+
+2. Navigate to the root folder and create a new ArduinoCLI directory, then enter it
+
+```sh
+cd STEP/STEP/ && mkdir ArduinoCLI && cd ArduinoCLI/
+```
+
+
+3. Download the Arduino-CLI for your preferred operating system from this source and place it inside the new folder:
+
+Linux:
+```sh
+wget https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_Linux_64bit.tar.gz
+```
+Windows:
+```sh
+wget https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_Windows_64bit.zip
+```
+macOS:
+```sh
+wget https://downloads.arduino.cc/arduino-cli/arduino-cli_latest_macOS_64bit.tar.gz
```

Comment:
Bare en kommentar: Det virker sjovt at inkludere macOS (og linux for den sags skyld) hvis vi ikke har tænkt os at kompilere vores compiler til de platforme (men det kan vi jo bare gøre)